src/mol-model/structure/model/model.ts

```diff
@@ -122,7 +122,10 @@ export namespace Model {
                 const symmetry = ModelSymmetry.fromCell(f.cell.size, f.cell.anglesInRadians);
                 ModelSymmetry.Provider.set(m, symmetry);
             }
-
+            if (oldIndex) {
+                // can I update here the indexPair Bond to take in account the periodicity?
```

Comment:
Is this were the periodicity of the bonds be handled ? or elsewhere ?

Reply:
How about setting a maxDistance when creating the IndexPairBonds and then use the dynamicBonds param for the structure